### PR TITLE
Trafodion 1822

### PR DESCRIPTION
**Pull request by gtapper**
Website updates. Please rebuild 1.3 and 2.0 documentation to pick up the official logo; the previous version turned out to be a screenshot where the blue was of the wrong hue. 

Thanks!

### CODE REVIEW

File: docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc [hunk 1]
```diff
@@ -196,7 +194,7 @@ You can monitor the SQ shell log file : /home/trafodion/apache-trafodion-1.3.0-i
 Startup time  0 hour(s) 1 minute(s) 9 second(s)
 Apache Trafodion Conversational Interface 1.3.0
 Copyright (c) 2015 Apache Software Foundation
->> initialize trafodion;
+>>
```
Comment (DaveBirdsall) [inline docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc]: Looks like we have a mystery command now that sqlci completed. 

I'm guessing you wanted to delete this prompt and also the "SQL opteration complete" message after it?

File: docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc [hunk 2]
```diff
@@ -470,7 +465,7 @@ You can monitor the SQ shell log file : /home/trafodion/apache-trafodion-1.3.0-i
...
```
Comment (DaveBirdsall) [inline docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc]: Same comment as above

**Comment by DaveBirdsall** [comment]
Looks good to me apart from one question above.

File: docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc (hunk 1 above)
Comment (gtapper) [inline docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc]: The command is piped into sqlci so it's not shown. It's very weird but I'm
documenting behavior, not changing it.

On Thu, Feb 25, 2016 at 5:10 PM, DaveBirdsall <notifications@github.com>
wrote:

> In docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc
> <https://github.com/apache/incubator-trafodion/pull/341#discussion_r54186985>
> :
>
> > @@ -196,7 +194,7 @@ You can monitor the SQ shell log file : /home/trafodion/apache-trafodion-1.3.0-i
> >  Startup time  0 hour(s) 1 minute(s) 9 second(s)
> >  Apache Trafodion Conversational Interface 1.3.0
> >  Copyright (c) 2015 Apache Software Foundation
> > ->> initialize trafodion;
> > +>>
>
> Looks like we have a mystery command now that sqlci completed.
>
> I'm guessing you wanted to delete this prompt and also the "SQL opteration
> complete" message after it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/341/files#r54186985>.
>



-- 
Thanks,

Gunnar
*If you think you can you can, if you think you can't you're right.*

Comment (DaveBirdsall) [inline docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc]: OK, thanks for the explanation.

**Comment by DaveBirdsall** [comment]
Looks good to me.

**Comment by gtapper** [comment]
Closing since we don't want source .ai files in the source tree.

**Pull request closed without merging**